scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -109,6 +110,11 @@ case class Args(
         "sourceroot. Defaults to current working directory if not provided."
     )
     sourceroot: Option[AbsolutePath] = None,
+    @Description(
+      "Absolute path passed to semanticdb with -P:semanticdb:targetroot:<path>. " +
+        "Used to locate semanticdb files. Default, Scalafix will try to locate semanticdb files under the classpath"
+    )
+    targetroot: Option[AbsolutePath] = None,
```

Comment:
targetroot makes sense as a scalac compiler option, but here it's an input, so I think it should be more explicit
```suggestion
    semanticdbRoot: Option[AbsolutePath] = None,
```

Reply:
or semanticTargetroot?
since we also have sourceroot which also specific to semanticdb options.

Reply:
Apologies, I didn't see that one! Interestingly, its default does not seem to be infered by the scalacOptions. I guess for symmetry, it makes sense to keep `targetRoot` then, even though I find both options a bit hard to understand for someone not familiar with scalameta. Resolving this

Reply:
I think it's better to name semanticdbTargetroot. I always had difficulties to understand sourceroot too.

Reply:
and I agree, it's weird we don't configure semanticdbSourceroot from scalacOptions.